### PR TITLE
update german translation

### PR DESCRIPTION
**Pull request by Somberland**
see https://github.com/DamascenoRafael/reminders-menubar/issues/109

### CODE REVIEW

File: reminders-menu-bar/Resources/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -35,7 +35,7 @@
 "appAppearanceMenu" = "Erscheinungsbild";
 "appAppearanceMoreOpaqueOptionButton" = "Hintergrund undurchsichtig";
 "appAppearanceMoreTransparentOptionButton" = "Hintergrund transparenter";
-"menuBarSettingsMenu" = "Menu bar"; // TODO: Translate
+"menuBarSettingsMenu" = "Menüleiste";
 "showMenuBarTodayCountOptionButton" = "Anzahl der offenen Erinnerungen in der Menüleiste anzeigen"; // TODO: Update this to be equivalent to "Show today count" (since this option will be inside MenuBarSettingsMenu).
```
Comment (DamascenoRafael) [inline reminders-menu-bar/Resources/de.lproj/Localizable.strings:39]: @Somberland can you please take a look at `showMenuBarTodayCountOptionButton`?

Comment (Somberland) [reply reminders-menu-bar/Resources/de.lproj/Localizable.strings]: missed that one, sorry :)

Comment (DamascenoRafael) [reply reminders-menu-bar/Resources/de.lproj/Localizable.strings]: No worries. Thanks for the update

**Comment by DamascenoRafael** [comment]
Thanks for your contribution 🎉

**Pull request merged**